packages/fluentui/docs/src/examples/components/Datepicker/Usage/index.tsx

```diff
@@ -43,7 +43,7 @@ const Usage = () => (
       description={
```

Comment:
Please let's move those examples to `Variants` block

Reply:
Done